scripts/run_network.py

```
@@ -0,0 +1,204 @@
+        return
+    print("\nShutting down all Meerkat nodes...")
+    for node_name, proc in processes:
+        if proc.poll() is None:
+            print(f"Stopping server '{node_name}' (PID: {proc.pid})...")
+            try:
+                proc.terminate()
+                proc.wait(timeout=2)
+            except subprocess.TimeoutExpired:
+                proc.kill()
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)